resharper/resharper-unity/src/CSharp/Feature/Services/QuickFixes/CallGraph/ExpensiveCodeAnalysis/ExpensiveCodeActionsUtil.cs

```diff
@@ -0,0 +1,7 @@
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.QuickFixes.CallGraph.ExpensiveCodeAnalysis
+{
+    public static class ExpensiveCodeActionsUtil
+    {
+        public const string MESSAGE = "Enable Expensive Analysis";
```

Comment:
As a context action text, I think this should be in sentence case. AIUI, this context action will treat the method declaration as the root of a performance critical context, so how about using: `Start performance critical context from here`?